tests/ci/build-pkg.yml

```diff
@@ -50,13 +50,7 @@ steps:
 
 - id: 'check source code formatting'
   name: 'gcr.io/redpandaci/builder'
-  entrypoint: 'bash'
-  args:
-  - -c
-  - |
-    if [[ "$_COMPILER" == "clang" ]]; then
-      ./build/venv/v/bin/vtools fmt all --check --conf=vtools/vtools/artifacts/ci/vtools-$_COMPILER-$_BUILD_TYPE.yml
-    fi
+  args: ['./build/venv/v/bin/vtools', 'fmt', 'all', '--check', '--conf=vtools/vtools/artifacts/ci/vtools-clang-$_BUILD_TYPE.yml']
```

Comment:
Would we be able to offload this to github actions (already done there) and not do linting at all here?
Although we want the internal build to be full on its own.

Reply:
yeah, I'd vote for not doing this here.

the deeper question that is burning in my mind is: how do we avoid having to maintain two set of software delivery pipelines 😩  . out of this PR's scope though